supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -7,3 +7,24 @@
 
 [[external-links]]
 == External links
+
+[[rh-kb-links]]
+== Linking to Red Hat Knowledgebase articles
+
+Guidelines:
+
+* Use the title of the Knowledgebase article for the link text, or use descriptive running text.
+* Call out that this is a Knowledgebase article when _not_ using running text.
+* When the link appears in *Additional resources*, put the article title first, followed by `(Red Hat Knowledgebase solution)`
```

Comment:
I like this approach. I would suggest dropping "solution" though -> "link:url[Article title] (Red Hat Knowledgebase)"

Reply:
I agree, I like this approach as well, and +1 to dropping "solution"

Reply:
@oraNod @bergerhoffer Done.